about.html

```diff
@@ -16,17 +16,21 @@
 <body>
     <header>
         <ul>
-            <li><img src="pics/icon/facebook.png" alt="Facebook"></li>
-            <li><img src="pics/icon/twitter.png" alt="Twitter"></li>
+            <li><img id="face" src="pics/icon/facebook.png" alt="Facebook"></li>
+            <li><img id="twi" src="pics/icon/twitter.png" alt="Twitter"></li>
             <li>English</li>
             <li>My page</li>
             <li>Logout</li>
```

Comment:
- [ ] Awesome implementing this in `index.html` file, kindly do the same in about.html file as well. **Note: The code reviewer actually pointed at about.html document.** 👍

Reply:
😸 solved, I forget the second page, tky for your advise